pkg/cmd/project/project.go

```diff
@@ -0,0 +1,31 @@
+package project
+
+import (
+	"fmt"
+	"github.com/MakeNowJust/heredoc/v2"
+	cmdList "github.com/OctopusDeploy/cli/pkg/cmd/project/list"
+	"github.com/OctopusDeploy/cli/pkg/constants"
+	"github.com/OctopusDeploy/cli/pkg/constants/annotations"
+	"github.com/OctopusDeploy/cli/pkg/factory"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdProject(f factory.Factory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "project <command>",
+		Aliases: []string{"proj"},
```

Comment:
we've not put an alias on a toplevel command before. 
@jbristowe what do you think about this? I'm very slightly leaning towards "let's not do that", but not concerned either way.

Reply:
Agreed. I do not wish to have aliases for top-level commands. Please remove the `env` alias as well.

Reply:
`env` is now removed, no other top-level commands have an alias defined